sql/share/errmsg-utf8.txt

```diff
@@ -7364,584 +8294,798 @@ skip-to-error-number 2000
 
 ER_CHANGE_MASTER_PASSWORD_LENGTH
-  eng "The password provided for the replication user exceeds the maximum length of 32 characters"
+        chi "为Replication User提供的密码超过32个字符的最大长度"
+        eng "The password provided for the replication user exceeds the maximum length of 32 characters"
 
 ER_USER_LOCK_WRONG_NAME 42000
+        chi "用户级锁名名称'%-.192s'不正确。"
         eng "Incorrect user-level lock name '%-.192s'."
 
 # Should be different from ER_LOCK_DEADLOCK since it doesn't cause implicit
 # rollback. Should not be mapped to SQLSTATE 40001 for the same reason.
 ER_USER_LOCK_DEADLOCK
-  eng "Deadlock found when trying to get user-level lock; try rolling back transaction/releasing locks and restarting lock acquisition."
+        chi "在尝试获得用户级锁时发现死锁;尝试回滚交易/释放锁定并重新启动锁定采集。"
+        eng "Deadlock found when trying to get user-level lock; try rolling back transaction/releasing locks and restarting lock acquisition."
 
 ER_REPLACE_INACCESSIBLE_ROWS
-  eng "REPLACE cannot be executed as it requires deleting rows that are not in the view"
+        chi "无法执行REPLACE，因为它需要删除不在视图中的行"
+        eng "REPLACE cannot be executed as it requires deleting rows that are not in the view"
 
 ER_ALTER_OPERATION_NOT_SUPPORTED_REASON_GIS
-  eng "Do not support online operation on table with GIS index"
-
+        chi "不要支持使用GIS索引的表中的在线操作"
+        eng "Do not support online operation on table with GIS index"
 # MariaDB extra error numbers starts from 4000
 skip-to-error-number 4000
 
 ER_COMMULTI_BADCONTEXT 0A000
+        chi "COM_MULTI无法返回给定上下文中的结果集"
         eng "COM_MULTI can't return a result set in the given context"
         ger "COM_MULTI kann im gegebenen Kontext keine Ergebnismenge zurückgeben"
         ukr "COM_MULTI не може повернути результати у цьому контексті"
 ER_BAD_COMMAND_IN_MULTI
+        chi "COM_MULTI不允许命令'%s'"
         eng "Command '%s' is not allowed for COM_MULTI"
         ukr "Команда '%s' не дозволена для COM_MULTI"
 ER_WITH_COL_WRONG_LIST
+        chi "使用列列表并选择字段列表具有不同的列计数"
         eng "WITH column list and SELECT field list have different column counts"
 ER_TOO_MANY_DEFINITIONS_IN_WITH_CLAUSE
+        chi "WITH条款中的元素太多了"
         eng "Too many WITH elements in WITH clause"
 ER_DUP_QUERY_NAME
+        chi "WITH子句重复查询名称%`-.64s"
         eng "Duplicate query name %`-.64s in WITH clause"
 ER_RECURSIVE_WITHOUT_ANCHORS
+        chi "没有元素'%s'递归的锚点"
         eng "No anchors for recursive WITH element '%s'"
 ER_UNACCEPTABLE_MUTUAL_RECURSION
+        chi "锚定表'%s'不可接受的相互递归"
         eng "Unacceptable mutual recursion with anchored table '%s'"
 ER_REF_TO_RECURSIVE_WITH_TABLE_IN_DERIVED
+        chi "物质化的衍生参考指向递归的WITH 表'%s'"
         eng "Reference to recursive WITH table '%s' in materialized derived"
 ER_NOT_STANDARD_COMPLIANT_RECURSIVE
+        chi "表'%s'R_WRONG_WINDOW_SPEC_NAME违反了递归定义的限制"
         eng "Restrictions imposed on recursive definitions are violated for table '%s'"R_WRONG_WINDOW_SPEC_NAME
 ER_WRONG_WINDOW_SPEC_NAME
```

Comment:
The `R_WRONG_WINDOW_SPEC_NAME` is a stray string (duplicated from the following line) that was probably discarded by the error message generation. It should not have been added to the Chinese translation. Fixed in 10.6 fae0ccad6ec391bf85c39070f784589d48e11951.